### PR TITLE
Update README.adoc

### LINKED ISSUE #4: Terminology Changes
Per issue #267 in the docs repo

### PR DESCRIPTION
**Pull request by t-koman**
Closes #4

### CODE REVIEW

**Comment by t-koman** [comment]
Hello,
I have signed the CLA
thanks,
Tom

IBM - WebSphere Application Server Information DevelopmentBuilding 707, Floor 1, Office H612455 South Road, Poughkeepsie, NY 12601T/L: 8-295-8668 PHONE: (845) 435-8668tkoman@us.ibm.com
 
 
----- Original message -----From: CLA assistant <notifications@github.com>To: kabanero-io/guide-using-kAppNav <guide-using-kAppNav@noreply.github.com>Cc: t-koman <tkoman@us.ibm.com>, Author <author@noreply.github.com>Subject: [EXTERNAL] Re: [kabanero-io/guide-using-kAppNav] Update README.adoc (#5)Date: Wed, Feb 12, 2020 7:39 PM 
Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our Contributor License Agreement before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us recheck it.

File: README.adoc [hunk 1]
```diff
@@ -83,13 +83,13 @@ image::/img/guide/guide-using-kAppNav-create-application.png[link="/img/guide/gu
 +
 . On the General tab, specify the application name and namespace.
 ** The **Name** value can be anything and does not need to match the name of the application.
-** The **Namespace** value must be the same namespace as the projects that make up the application.
+** The **Namespace** value MUST BE the same namespace as the projects that make up the application.
```
Comment (jantley-ibm) [inline README.adoc]: Change
MUST BE
to
must be

**Comment by jantley-ibm** [comment]
@t-koman let me know when updated and I will merge -- thanks

**Comment by t-koman** [comment]
The update has been made...... MUST BE ---- to must be

**Pull request merged**